configure.ac

```diff
@@ -21,23 +21,20 @@ AC_SUBST(PKG_CONFIG_EXTRA_PATH, "")
 AC_SUBST(EXTRA_LD_FLAGS, "")
 
 AC_MSG_CHECKING([operating system])
-case "$host" in
-*-linux*)
-	AC_MSG_RESULT([$host (Linux)])
-	hidapi="hidapi-libusb"
-
-	#link with realtime lib on linux for clock_gettime
-	AC_SUBST(EXTRA_LD_FLAGS, "-lrt -lpthread")
-	;;
-*-freebsd*)
-	AC_SUBST(PKG_CONFIG_EXTRA_PATH, "libdata/")
-	;;
-esac
+AS_CASE(["$host"],
+	[*-linux*],
+		[AC_MSG_RESULT([$host (Linux)])
+		hidapi="hidapi-libusb"
+
+		#link with realtime lib on linux for clock_gettime
+		AC_SUBST(EXTRA_LD_FLAGS, "-lrt -lpthread")],
+	[*-freebsd*],
+		[AC_SUBST(PKG_CONFIG_EXTRA_PATH, "libdata/")])
 
 # Oculus Rift Driver
 AC_ARG_ENABLE([driver-oculus-rift],
-        [AS_HELP_STRING([--enable-driver-oculus-rift],
-                [enable building of Oculus Rift driver (default y)])],
+        [AS_HELP_STRING([--disable-driver-oculus-rift],
+                [disable building of Oculus Rift driver [default=yes]])],
```

Comment:
So the logic of these makes me confused. By default "driver-oculus-rift" is yes?